HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -397,6 +412,17 @@ list_models(){
 			REPOS+=("$(dirname "$MODEL")")
 		fi
 	done
+	for ENSEMBLEMODEL in ${ENSEMBLEMODELS[@]}; do
```

Comment:
I think this logic will fail in the case of nested ensemble models (which are allowed by Triton, as far as I know).

Maybe simpler just to call `edit_model` recursively on submodels, whenever an ensemble model is found? (`edit_model` would need to encompass all the operations in this loop, to handle cases with or without `INSTANCES` specified.)

Reply:
Yes, I made this recursive, thank you for the suggestion!